lib/puppet/provider/user/hpux.rb

```diff
@@ -1,7 +1,13 @@
 Puppet::Type.type(:user).provide :hpuxuseradd, :parent => :useradd do
   desc "User management for HP-UX. This provider uses the undocumented `-F`
     switch to HP-UX's special `usermod` binary to work around the fact that
-    its standard `usermod` cannot make changes while the user is logged in."
+    its standard `usermod` cannot make changes while the user is logged in.
+<<<<<<< HEAD
```

Comment:
This is a merge conflict that needs to be resolved. Git adds these when it can't automatically merge based on commit history. You should be able to remove lines 5 and 7-9.